### PR TITLE
update logrus import path

### PR DESCRIPTION
**Pull request by jtlisi**
https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276

Due to the recent change of Sirupsen->sirupsen this import path must be updated in the client-lib.
I will make a similar PR to the main snap project.

### CODE REVIEW

**Comment by Yajuan-Shi** [comment]
LGTM
First code review about this pull request has pass and compile, test be executed.

File: glide.lock [hunk 1]
```diff
@@ -38,11 +46,12 @@ imports:
   subpackages:
   - googleapis/rpc/status
 - name: google.golang.org/grpc
-  version: b8669c35455183da6d5c474ea6e72fbf55183274
+  version: f92cdcd7dcdc69e81b2d7b338479a19a8723cfa3
```
Comment (sandlbn) [inline glide.lock]: Can we change it back to b8669c35455183da6d5c474ea6e72fbf55183274 to match snap version https://github.com/intelsdi-x/snap/blob/master/glide.lock

**Comment by IzabellaRaulin** [comment]
@jtlisi, I created PR to your branch addressing the last comment and including some "Notice" section about what issue might occur and how to resolve it during updating a plugin to the latest snap-plugin-lib-go. Let me know what do you think about it - https://github.com/jtlisi/snap-plugin-lib-go/pull/1.

**Comment by jtlisi** [comment]
@IzabellaRaulin Thanks for the PR I think that really clears things up for snap-client users!!

**Pull request merged**